subprojects/parseq/build.gradle

```diff
@@ -34,6 +36,9 @@ dependencies {
   testCompile project(':parseq-test-api')
   testCompile group: "org.testng", name: "testng", version: "6.9.9"
   testCompile group: "org.slf4j", name: "slf4j-log4j12", version: "1.7.25"
+
+  // TODO: remove in parseq 5
```

Comment:
How can you remove in parseq 5? Will the same problem persist?

Reply:
Answered this offline, but I'll repeat here for visibility. Yes, removing the configuration in parseq 5 will cause the same problem for anyone who's still consuming the `testArtifact` configuration. However, this gives us a window of support between parseq 4 and 5 where support for both usage patterns is supported, which makes it easier for us to migrate all internal consumers. By the time we want to bump to parseq 5, all internal consumers will have been migrated away from the `testArtifact` configuration, so the impact will be much more manageable.

Reply:
Perhaps it would've been wiser to bump the minor version first (since the configuration wouldn't have been removed yet), then bump the major version to remove the configuration. Hindsight is always 20/20 though... Anyway, the goal here is to provide a window of support for both options. We want to remove the `testArtifact` configuration as soon as we can though.